integration/src/main/java/scripts/client/DataRepoClient.java

```diff
@@ -0,0 +1,49 @@
+package scripts.client;
+
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.testrunner.common.utils.AuthenticationUtils;
+import bio.terra.testrunner.runner.config.ServerSpecification;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+public class DataRepoClient extends ApiClient {
+
+  // Required scopes for client tests include the usual login scopes and GCP scope.
+  public static final List<String> TEST_USER_SCOPES =
+      List.of("openid", "email", "profile", "https://www.googleapis.com/auth/cloud-platform");
```

Comment:
Should we make this configurable generally and select them for the test? It may be useful for our swagger page too.

Reply:
I agree, this should be consolidated and not duplicated across different places. I'll look at cleaning that up, if it's too much work for now I'll make a ticket for it.

Reply:
I've updated this to use a constant from terra-common-lib.